src/parseCompilationText.ts

```diff
@@ -0,0 +1,21 @@
+function parseCompilationText(compilation_text: string) {
+    // Manually parse Compilation text into each part
+    const compilation_text_split = compilation_text.split("Circuit")
+    const input_circuit = compilation_text_split[1].slice(2)
+
+    const pauli_rotations_split = compilation_text_split[2].split(":")
+    const compilationTextJson = {
+        input_circuit: input_circuit,
+        circuit_after_pauli_rotations: pauli_rotations_split[1].slice(1, -1),
+        circuit_after_litinski: "",
+    }
+    console.log("PIXZZA11111:", compilationTextJson)
```

Comment:
Remove `console.log`.

Reply:
Thanks. That is embarrassing. Makes me think of how auto graders would give you zero points if your submission contained commented code, lol.

Reply:
haha i didn't notice those print statements at all until pressing F12 on my browser tho.

Reply:
At least it's not obscene...

Reply:
PIZZA is my go to console.log. More Z's = more important.